source/client-side-encryption/client-side-encryption.rst

```diff
@@ -838,8 +844,27 @@ Identifies a key vault collection document by 'keyAltName'.
 
 algorithm
 ^^^^^^^^^
-The string "AEAD_AES_256_CBC_HMAC_SHA_512-Deterministic" or
-"AEAD_AES_256_CBC_HMAC_SHA_512-Random"
+One of the strings:
+- "AEAD_AES_256_CBC_HMAC_SHA_512-Deterministic"
+- "AEAD_AES_256_CBC_HMAC_SHA_512-Random"
+- "IndexedEquality"
+- "Unindexed"
+
+The result of explicit encryption with the "IndexedEquality" algorithm must be processed by the server to insert or query. Drivers MUST document the following behavior:
+
+   To insert or query with an "IndexedEquality" encrypted payload, use a ``MongoClient`` configured with ``AutoEncryptionOpts``.
+   ``AutoEncryptionOpts.bypassQueryAnalysis`` may be true. ``AutoEncryptionOpts.bypassAutoEncryption`` must be false.
+
+contentionFactor
+^^^^^^^^^^^^^^^^
+contentionFactor only applies when algorithm is "IndexedEquality".
```

Comment:
Where does libmongocrypt pick a random partition on insert like: https://github.com/mongodb/mongo/commit/52e26913ceedc6cf87780922bae90f12d9cb62ff

Reply:
Great question. On rereading I do not think libmongocrypt is picking a random partition on insert. Will fix this as part of  [MONGOCRYPT-426](https://jira.mongodb.org/browse/MONGOCRYPT-426).